config/default.yml

```diff
@@ -0,0 +1,58 @@
+AvoidOutlineForSingleExample:
+    Enabled: true
+AvoidPeriod:
+    Enabled: true
+AvoidScripting:
+    Enabled: true
+BackgroundDoesMoreThanSetup:
+    Enabled: true
+BackgroundRequiresMultipleScenarios:
+    Enabled: true
+BadScenarioName:
+    Enabled: true
+BeDeclarative:
+    Enabled: true
+FileNameDiffersFeatureName:
+    Enabled: true
+MissingExampleName:
+    Enabled: true
+MissingFeatureDescription:
+    Enabled: true
+MissingFeatureName:
+    Enabled: true
+MissingScenarioName:
+    Enabled: true
+MissingTestAction:
+    Enabled: true
+MissingVerification:
+    Enabled: true
+InvalidFileName:
+    Enabled: true
+InvalidStepFlow:
+    Enabled: true
+RequiredTagsStartsWith:
+    Enabled: false
```

Comment:
See above comment.  The idea is that there are multiple types of TagCollection linters so it didn't make sense to always have an enabled default since, at some point, those linters would easily conflict with each other.